tests/integration/project/cmd_project_test.go

```diff
@@ -38,6 +39,44 @@ var _ = Describe("odo project command tests", func() {
 		})
 	})
 
+	Context("Should be able to delete a project with --wait", func() {
+		var projectName string
+		JustBeforeEach(func() {
+			projectName = helper.RandString(6)
+		})
+
+		It("--wait should work with deleting a project", func() {
+
+			// Create the project
+			helper.CmdShouldPass("odo", "project", "create", projectName)
+
+			// Delete with --wait
+			output := helper.CmdShouldPass("odo", "project", "delete", projectName, "-f", "--wait")
+			Expect(output).To(ContainSubstring("Waiting for project to be deleted"))
+
+			// Check that it's NOT there..
+			output = helper.CmdShouldPass("odo", "project", "list")
```

Comment:
This seems to run into a flake due to the explanation given in the note of the DeleteProject() function
https://travis-ci.com/openshift/odo/jobs/259258747#L3847

I guess we may need to add a sleep timeout?